### ISSUE TITLE
Tarantula Hawk taming is buggy

### ISSUE THREAD

**Issue by mineralss** [Alex-the-666/AlexsMobs#997]
Version: alexsmobs-1.17.2
I tamed a Tarantula Hawk, however it would not follow me (it did sit when commanded however). After a little while of searching after it flew off somewhere, I located it. I previously named it with a nametag before taming, so I knew for certain it was the same Tarantula Hawk. However, right-clicking on the mob did nothing, not even the white text appeared. I then hit it, and it became aggressive towards me. 
tl;dr tar hawk following doesn't work + it untamed itself

**Comment by mineralss** [comment]
Tested in a creative world, magically working now. I dont even know anymore....
